src/main/java/net/sf/jabref/importer/UnlinkedFilesCrawler.java

```diff
@@ -60,12 +61,24 @@ public CheckableTreeNode searchDirectory(File directory, UnlinkedPDFFileFilter f
             return null;
         }
 
-        List<File> files = Arrays.asList(directory.listFiles(ff));
+        File[] filesArray = directory.listFiles(ff);
```

Comment:
Would make sense, but also requires rewriting `UnlinkedPDFFileFilter`, so I will not get into this in the current PR.